node/src/index-send-xrp-to-payid.js

```diff
@@ -0,0 +1,70 @@
+const open = require('open')
```

Comment:
This demo already exists in index-xpring.js

Reply:
deleted this one since it is duplicative